setup.sh

```diff
@@ -162,6 +159,27 @@ function confirm_complete_removal()
   fi
 }
 
+# This function moves the folders from the old directory structure to
+# folders of the new one.
+function legacy_folders()
+{
+  local prefix="$HOME/.local"
+
+  if [[ -d "$prefix/share/doc" && -f "$prefix/share/man/kw.rst" &&
+    -d "$prefix/share/sound/kw" && -d "$prefix/etc/kw" &&
+    -d "$HOME/.kw" ]]; then
+
+    mv "$HOME/.kw" "$datadir"
+    mv "$prefix/share/doc" "$docdir"
+    mkdir -p "$mandir"
+    mv "$prefix/share/man/kw.rst" "$mandir/"
+    mv "$prefix/share/sound/kw" "$sounddir"
+    mv "$prefix/etc/kw" "$etcdir"
+    rmdir --ignore-fail-on-non-empty "$prefix/share/man" "$prefix/share/sound" \
+      "$prefix/etc"
+  fi
```

Comment:
Technically, if it is a fresh installation, you don't need to move things around; you just need to remove the old code. The only thing you might want to move is the file under `~/.kw/`.